### PR TITLE
added signature_latte

### PR DESCRIPTION
**Pull request by CPranaviReddy**

### CODE REVIEW

**Review by Rajawardhan — COMMENTED** [review]
Modify the Menu file by adding the reference to the new signataure_latte into menu.

**Comment by CPranaviReddy** [comment]
> Modify the Menu file by adding the reference to the new signataure_latte into menu.

made changes accordingly

**Review by Rajawardhan — COMMENTED** [review]
Add signature_latte to menu file

**Comment by CPranaviReddy** [comment]
> Add signature_latte to menu file

File: cafe_branch2_menu.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <html>
 <title>Joe’s café - branch2 menu</title>
+<ul>
+<li>signature_latte</li>
```
Comment (Rajawardhan) [inline cafe_branch2_menu.md]: Has special character at cafe, please correct it.

File: cafe_branch1_menu.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <html>
-<title>Joe’s café - branch1 menu</title>
+<title>Joe’s cafe - branch1 menu</title>
+ <ul>
+   <li> Signature_latte   </li>
+ </ul>
```
Comment (Rajawardhan) [inline cafe_branch1_menu.md:5]: Please remove the underscore in the Signature_latte

File: cafe_branch2_menu.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
 <html>
-<title>Joe’s café - branch2 menu</title>
+<title>Joe’s cafe - branch2 menu</title>
+<ul>
+<li>signature_latte</li>
```
Comment (Rajawardhan) [inline cafe_branch2_menu.md]: Please remove the underscore in the Signature_latte

**Review by Rajawardhan — COMMENTED** [review]
Please check updated comments for content change

File: cafe_branch2_menu.md (hunk 1 above)
Comment (CPranaviReddy) [reply cafe_branch2_menu.md]: updated

**Pull request merged**